### ISSUE TITLE
Twitter stopped working altogether

### ISSUE THREAD

**Issue by LRPinheiro** [satsukitv/twitter-block-chain#88]
Hello,

On my first attempt the extension worked (though in a weird way, because every time I came back to the tab to see how many accounts had been blocked, the number was lower than the number that was appearing there when I had last seen the tab). However, now I can't access anything on Twitter. Nothing works. Nothing. I just get a blank page saying that something went wrong. What am I missing?

**Comment by LRPinheiro** [comment]
I just checked: it does work on Firefox, though.

**Comment by vasilenka** [comment]
I'm having the same issue here

UPDATE: I solved it by logging out of Twitter

**Comment by ghost** [comment]
I am also having this issue on Google Chrome, can't log in to Twitter, just says:
![help](https://user-images.githubusercontent.com/95390160/144338244-2fdfab12-85ab-4725-8db9-9405959c206c.PNG)

So I uninstalled it, thinking maybe unstalling would fix the issue, but I cannot use Twitter on Chrome anymore. Is there any way to fix this?

EDIT: FIXED BY DELETING COOKIES, SORRY FOR WASTING YOUR TIME.

**Comment by PammWhittaker** [comment]
That didn't work for me. It always locked up if I tried more than 100 names. But now it won't even do 10 :(

I use it on Chrome, there's a Firefox addon? I can't find it.. thanks

**Comment by Super-Dope999** [comment]
Yeah I get that everyone is saying you can just log out or delete cookies and browsing history but then it doesn’t block the people you want to block. They need to get in here and fix it.
